### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by cristianoc72**
Bump dependencies and improve code coverage.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,16 +27,36 @@
...
-		"symfony/config" : "~2|~3|^4",
-		"symfony/yaml" : "~2|~3|^4"
+		"php" : ">=7.2",
+		"symfony/config" : "^5.0",
+		"symfony/yaml" : "^5.0"
 	},
 	"require-dev" : {
-		"phpunit/phpunit" : "^5.7",
-		"sami/sami" : "~3|~4"
+		"phpunit/phpunit": "^9.0",
```
Comment (gossi) [inline composer.json]: just `^8.0` for now.

Comment (cristianoc72) [reply composer.json]: Sorry. I didn't see that composer installs 9.0 by default.

File: src/ProfileDefinition.php [hunk 2]
```diff
@@ -1,301 +1,305 @@
...
-		$rootNode = $treeBuilder->root('formatter');
-
-		$this->addIndentationSection($rootNode);
-		$this->addBracesSection($rootNode);
-		$this->addWhitespaceSection($rootNode);
-		$this->addNewlinesSection($rootNode);
-		$this->addBlanksSection($rootNode);
+	public function getConfigTreeBuilder(): TreeBuilder {
+		$treeBuilder = new TreeBuilder('formatter');
+		/** @var NodeDefinition $node */
```
Comment (gossi) [inline src/ProfileDefinition.php]: I can see no variable `$node` in the current scope? Or is github diff view tricking me? What's this for?

Comment (cristianoc72) [reply src/ProfileDefinition.php]: Sorry, this old comment is here for a mistake. Fixed.

**Pull request merged**